src/features/lightspeed/api.ts

```diff
@@ -215,14 +219,15 @@ export class LightSpeedAPI {
       return response.data;
     } catch (error) {
       const err = error as AxiosError;
-      vscode.window.showErrorMessage(retrieveError(err));
+      const mappedError: IError = mapError(err);
+      vscode.window.showErrorMessage(mappedError.message ?? UNKNOWN_ERROR);
```

Comment:
I'm now seeing this error popup as a seated user. It won't make sense to them, and we also don't ever want to pop a visible error for implicit feedback problems (inline suggestion, ansible content).
![image](https://github.com/ansible/vscode-ansible/assets/3698180/c9af708d-af55-4157-b76f-9b6cab72c949)

Reply:
What did you do to see this?